src/deploy/osp_deployer/settings/sample.properties

```diff
@@ -31,6 +31,14 @@
 
         "private_api_ip":"192.168.140.13"
     },
+    {
+        "is_powerflexgw: "true",
+        "hostname": "powerflexgw",
+        "root_password": "xxxxxxxxxxx",
+        
+        "public_api_ip": "192.168.190.10",
+        "storage_ip": "192.168.170.10"
+    },
     {
```

Comment:
Would not innclude this in default sample, only in powerflex sample

Reply:
Fixed.